README.md

```diff
@@ -17,39 +17,57 @@ Build micro frontends that coexist and can each be written with their own framew
 - Write code using a new framework, without rewriting your existing app
 - Lazy load code for improved initial load time.
 
+## Documentation
```

Comment:
as discussed, let's have this readme not be a duplicate of what's on the website